Notify/Notify/Notify/ViewModels/WelcomePageViewModel.cs

```diff
@@ -53,7 +55,7 @@ private async void onLoginClicked()
             bool debugAutoLogin = true;
             IsBusy = true;
             
-            manageLocationTracking();
+            locationService.ManageLocationTracking();
```

Comment:
I think this should only be called after the user has logged in, and not before we're checking if he's authenticated.

Reply:
Right